music_assistant/providers/lrclib/__init__.py

```diff
@@ -110,6 +112,16 @@ async def get_track_metadata(self, track: Track) -> MediaItemMetadata | None:
             )
             return None
 
+        # check, if the track comes from any filesystem provider, and might have stored
+        # local lyrics
+        if (filesystem_provider := self.mass.get_provider(track.provider)) and isinstance(
+            filesystem_provider, LocalFileSystemProvider | SMBFileSystemProvider
+        ):
+            if lrc_lyrics := await filesystem_provider._get_lrc_data(track):
+                metadata = MediaItemMetadata()
+                metadata.lrc_lyrics = lrc_lyrics
+                return metadata
+
```

Comment:
No, this is not allowed this way as it mixes up responsibilities.
Parsing the metadata from the filesystem is contained in the filesystem, not here.